todo.sh

```diff
@@ -1485,14 +1485,22 @@ note: PRIORITY must be anywhere from A to Z."
 "listaddons" )
     if [ -d "$TODO_ACTIONS_DIR" ]; then
         cd "$TODO_ACTIONS_DIR" || exit $?
+        VALID=0
         for action in *
         do
             if [ -f "$action" ] && [ -x "$action" ]; then
                 echo "$action"
+                VALID+=1
```

Comment:
To increment, you need to use `((VALID+=1))` (or `let VALID+=1`); you're lucky that it works already.

Reply:
It didn't actually work as expected, as I discovered; the digit was just put after the variable content.